docker/dependencies/docker-compose.prometheus.yml

```diff
@@ -3,10 +3,28 @@
 version: "3.5"
 
 services:
+  # Use http://localhost:9090/ to access Prometheus.
   prometheus:
     image: prom/prometheus
     container_name: temporal-prometheus
-    ports:
-      - "9090:9090"
-    networks:
-      - temporal-dependencies-network
+    volumes:
+      - ./prometheus:/etc/prometheus
+    # Linux
+    network_mode: host
+    # darwin/Windows
```

Comment:
I don't want to create another version for Mac, as it is already advanced scenario, and people should be able to figure out it themselves.